src/illuminatio/test_orchestrator.py

```diff
@@ -1,6 +1,12 @@
+"""
+file containing all utilities to interact with test case related kubernetes resources
```

Comment:
Start with a capital letter to match the rest of the docstrings (see: https://www.python.org/dev/peps/pep-0257/)